cves/kernel/CVE-2013-1958.yml

```diff
@@ -1,483 +1,513 @@
+    that outsiders to this project would not understand. Technology like "regular
+    expressions" is fine, and security phrases like "invalid write" are fine to
+    keep too.
+
+    Your target audience is people just like you before you took any course in
+    security
+description: |
+    "The Linux kernel before version 3.8.6 contained a flaw in how it handled
+    setting UNIX domain socket process ID (PID) permissions during the creation
+    of new user and PID namespaces. A race condition existed where a local
+    attacker could temporarily gain elevated privileges in a new user namespace
+    to create sockets with PIDs that improperly mapped to the global PID
+    namespace. This allowed unauthorized access to domain sockets not belonging
+    to the attacker's process. The issue was addressed by properly checking
+    capabilities relative to the PID namespace when setting socket PIDs, closing
+    the temporary privilege escalation window."
 bounty_instructions: |
-  If you came across any indications that a bounty was paid out for this
-  vulnerability, fill it out here. Or correct it if the information already here
-  was wrong. Otherwise, leave it blank.
+    If you came across any indications that a bounty was paid out for this
+    vulnerability, fill it out here. Or correct it if the information already here
+    was wrong. Otherwise, leave it blank.
 bounty:
-  amt:
-  announced:
-  url:
+    amt:
+    announced:
+    url:
 reviews: []
 bugs_instructions: |
-  What bugs are involved in this vulnerability?
+    What bugs are involved in this vulnerability?
 
-  Please list bug IDs to https://bugzilla.kernel.org/
+    Please list bug IDs to https://bugzilla.kernel.org/
 
-  Bug ID's can appear in several places:
-    * Mentioned in commit messages
-    * Mentioned in mailing list discussions
-    * References from NVD entry
-    * Various other places
+    Bug ID's can appear in several places:
+      * Mentioned in commit messages
+      * Mentioned in mailing list discussions
+      * References from NVD entry
+      * Various other places
 bugs: []
 fixes_instructions: |
-  Please put the commit hash in "commit" below.
+    Please put the commit hash in "commit" below.
 
-  This must be a git commit hash from the systemd source repo, a  40-character
-  hexademical string/
+    This must be a git commit hash from the systemd source repo, a  40-character
+    hexademical string/
 
-  Place any notes you would like to make in the notes field.
+    Place any notes you would like to make in the notes field.
 fixes:
-- commit:
-  note:
-- commit:
-  note:
-- commit: 92f28d973cce45ef5823209aab3138eb45d8b349
-  note: |
-    Taken from NVD references list with Git commit. If you are
-    curating, please fact-check that this commit fixes the vulnerability and replace this comment with 'Manually confirmed'
+    - commit: 92f28d973cce45ef5823209aab3138eb45d8b349
+      note: "Manually confirmed"
 vcc_instructions: |
```

Comment:
I would recommend adding some notes to the commits describing briefly what they break to be considered a vcc. With so many contributing commits, I suspect a couple may not actually be too relevant.